Parse/src/main/java/com/parse/ParseApacheHttpClient.java

```diff
@@ -212,7 +212,7 @@ public ParseApacheHttpClient(int socketOperationTimeout, SSLSessionCache sslSess
   private static class ParseApacheHttpEntity extends InputStreamEntity {
     private ParseHttpBody parseBody;
 
-    public ParseApacheHttpEntity(ParseHttpBody parseBody) {
+    public ParseApacheHttpEntity(ParseHttpBody parseBody) throws IOException {
```

Comment:
Since parseBody.getContent() will try to create a fileStream, so we need to throw IOException here.